### PR TITLE
Fixed python interpreter version for tests

### PR DESCRIPTION
**Pull request by dbecvarik**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ test-py27: prepare
 	tox -e py27 -- tests
 
 test-py34: prepare
```
Comment (goldmann) [inline Makefile]: Name of the step should be changed too :)

**Comment by dbecvarik** [comment]
amended

**Comment by goldmann** [comment]
You need to update also: https://github.com/jboss-dockerfiles/dogen/blob/develop/circle.yml#L21

**Comment by dbecvarik** [comment]
hmmm so lets close this - I'll fix it in dogen-ng branch - I totaly forgot about circle.yaml fail

**Pull request closed without merging**